### PR TITLE
Add collapse molecule

### PR DESCRIPTION
**Pull request by dmarcoux**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *obs-patterns* ready!

Built with commit 2a97bcc6e30f417996e4f5f33786ebbfe392f7a5

https://deploy-preview-40--obs-patterns.netlify.com

File: _posts/molecules/2018-11-26-collapse.html
Comment (Ana06) [inline _posts/molecules/2018-11-26-collapse.html]: `request{ href: "/request/show/1" }` is not really related to collapse, so I would remove it to remove complexity (same everywhere)

File: _posts/molecules/2018-11-26-collapse.html
Comment (Ana06) [inline _posts/molecules/2018-11-26-collapse.html]: same here you can just write element1 (same everywhere)

File: _posts/molecules/2018-11-26-collapse.html
Comment (Ana06) [inline _posts/molecules/2018-11-26-collapse.html]: I would put also something more generic here. For example: 

```
%span.collapser See less elements
%span.expander See more elements
```

**Review by Ana06 — CHANGES_REQUESTED** [review]
I would add more details more in the commit description (why is that needed?)

**Comment by dmarcoux** [comment]
Somehow, it's not working (both icons are displayed)

**Comment by Ana06** [comment]
@dmarcoux 

> Somehow, it's not working (both icons are displayed)

haven't you forgotten the css? :thinking:

**Comment by dmarcoux** [comment]
The CSS from OBS is included [here](https://github.com/openSUSE/obs-patterns/blob/6564f5a6e590c1b5a7648fcf315cd5f03c2f0818/_includes/head.html#L10). I think there's an issue with CORS.

**Comment by dmarcoux** [comment]
I realized what the issue is. The CSS and JS needed for this to work is in the staging-workflow branch.

**Comment by Ana06** [comment]
@dmarcoux 

> I realized what the issue is. The CSS and JS needed for this to work is in the staging-workflow branch.

That is what I meant, you could add it either here or in master, right? Or we just wait until staging workflow is in master.

**Review by Ana06 — COMMENTED** [review]
Apart from the missing css, LGTM

**Comment by dmarcoux** [comment]
I think I will just wait for the staging-workflow branch to be merged on master. I sent this PR to be sure to not forget about adding the collapse molecule.

**Review by bgeuken — APPROVED** [review]

**Comment by dmarcoux** [comment]
@openSUSE/obs-frontend Please have a look.

The staging workflow branch has been merged in OBS, this now works. I also pushed some changes to display the collapsed/expanded version of the collapse molecule.

**Review by Ana06 — APPROVED** [review]

**Review by Ana06 — CHANGES_REQUESTED** [review]
I didn't want to approve it :see_no_evil: It seems something is wrong with the Javascript. It crasses when clicking `See less elements`

**Comment by dmarcoux** [comment]
I don't know why because it works in OBS, but not in this project... Something has to be different.

**Pull request closed without merging**
